crypto/Kconfig

```diff
@@ -51,6 +51,20 @@ config CC3XX_ATOMIC_LOCK
 	help
 	  Using atomic operations is the fastest way to ensure mutually exclusive
 	  access to the ARM CryptoCell hardware.
+
+	  Warning: If this configuration is set, every execution requiring use of
+	  the ARM CryptoCell hardware must happen in the same priority. Calling into
+	  mbed TLS APIs from a higher priority while an ongoing operation will lead
+	  to undefined behavior. It is highy recommended to to do all cryptographic
+	  operations in one single thread if this configuration is set.
+
+config CC3XX_HW_MUTEX_LOCK
```

Comment:
This needs a depends to ensure this functionality is only settable in context of nRF9160 and nRF5340 (app core)

This feature is unavailable in nRF52840